README.md

```diff
@@ -76,6 +76,36 @@ Before coding, you need to install Django in your virtual environment:
 pip install django
 ```
 
+##    Connecting to the OMDB API
+To fetch movie data from OMDB, you need to create an API key. Follow these steps:
+1. Go to the OMDB API website: https://www.omdbapi.com
+2. Click on the API Key tab.
+3. Scroll down to the Pricing section and select the Free or Paid plan depending on your usage.
+4. Sign up for an account using your email.
+5. Verify your email and log in.
+6. Navigate to the API Key section in your profile.
+7. Copy your generated API key.
+
+###    Adding the OMDB API Key to Your Project
+1. Create a .env file in the root directory of your project:
+```bash
+touch .env
+```
+2. Add your OMDB API key to the .env file:
+```bash
+OMDB_API_KEY=your_api_key_here
+```
+3. In your Django settings or configuration file, load the API key from the .env file:
+ ```bash
```

Comment:
This is already present in code. pls remove this step.

Reply:
okay mam

Reply:
![Screenshot 2024-10-18 155624](https://github.com/user-attachments/assets/3459ab33-eb8c-4f21-b431-15c8e1b82c5a)
is this fine now?

Reply:
We have defined the view code as API_KEY = os.getenv('API_KEY') so the name has to be chaged.

Reply:
![Screenshot 2024-10-18 161625](https://github.com/user-attachments/assets/7e84a7fe-c75f-4435-9df1-bc038a87434b)

Reply:
Since in the code we have used API_KEY instaed of OMDB_API_KEY. Can you change it to API_KEY = secret_key?

Reply:
LGTM

Reply:
may i make pull request now?

Reply:
Sure but also do a pull request so that there arent merge conflicts.

Reply:
@eleensmathew i tried but same pr me merge ho rha h